packages/dockest/src/utils/execaWrapper.ts

```diff
@@ -10,7 +10,7 @@ interface Opts {
   execaOpts?: SyncOptions<string>
 }
 
-export const execaWrapper = async (
+export const execaWrapper = (
```

Comment:
@erikengervall I changed this to a sync method because this function actually does nothing async. Since `packages/dockest/src/utils/getRunMode.ts` is sync (and most code that depends upon it).

If you want to keep it async we will have to make another wrapper for sync code (e.g. `packages/dockest/src/test-helper/index.ts` to `packages/dockest/src/utils/getOpts.ts`)

Reply:
Ah, it was probably just a mistake on my part! Thanks!